content/en/blog/_posts/2021-04-29-SIG-Usability-Spotlight.md

```diff
@@ -0,0 +1,68 @@
+
+A. New contributors can get started by shadowing one of the user interviews. Going through user interview transcripts and making analysis on user interviews. There are additionally plenty of opportunities in survey design background. 
+
+Sig-Usability is also open to new project ideas. If you have an idea, we’ll do what we can to fulfil that idea. There are also regular SIG Meetings for people to attend and ask their questions live. The meetings are also recorded for those who may not be able to attend. As always, you can reach out to us on Slack as well.
+
+### Q. What does the survey include?
+
+A. In simple terms, the survey includes information related to what are people doing with Kubernetes. We are looking for behaviours amongst users, examples include the trends of learning a new deployment system, error messages faced by the users, end-to-end tasks, etc. 
+
+One of our goals is to standardise the responses accordingly. The ultimate goal being analysis the survey responses and looking for under-served jobs, areas with high importance but least satisfaction.
+
+### Q. Are there any particular skills you’d like to recruit for? What skills are contributors to SIG-usability likely to learn?
+
+A. Although contributing to Sig-Usability does not have any pre-requisites as such, but experience with user research, qualitative research, prior experience with how to conduct an interview would be a great plus point. Along with that, quantitative research, like survey design, screening etc. are also helpful and some things that we expect the contributors to learn.
+
+
+### Q. What are you getting positive feedback on, and what’s coming up next for SIG-usability?
+
+a. We have had new members joining and coming to monthly meetings regularly and showing interests in becoming a contributor and helping the community. We have also had a lot of people reach out to us via Slack showcasing their interest in the sig. 
+
+Currently, we are focused on finishing this study which is our project for this year and we are always happy to have new contributors join us.
+
+### Q: Any closing thoughts/resources you’d like to share?
+
+A. We love meeting new contributors and perceive how we can assist them with investigating different spaces of the Kubernetes project. We will work with and team up with other SIGs on giving end-user channels for demonstrating input on their experience, studies and other criticism get-togethers around project convenience, documentation, plan components and language rules for Kubernetes concepts, recognizing opportunities for user research and best strategies to accomplish those.
+
+Here are some resources for you to get started:
+- [GitHub](https://docs.google.com/document/d/17zSSDRYoVySbxev2_9r1x89YzFtRH-5DUrz-Q3VAGuo/edit)
+- [Mailing list](https://groups.google.com/g/kubernetes-sig-usability)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fusability)
+- Slack: #sig-usability
```

Comment:
You can link to the slack channel as well: https://kubernetes.slack.com/archives/CLC5EF63T